webapp/components/PasswordReset/Request.spec.js

```diff
@@ -18,7 +18,7 @@ describe('Request', () => {
         success: jest.fn(),
         error: jest.fn(),
       },
-      $t: jest.fn(),
+      $t: jest.fn(t => t),
```

Comment:
we just ran into this issue today, on `ReportModal.spec.js`... would love to refactor all the `$t: jest.fn()` like this

Reply:
I know.. we learn everyday. From now on I will always check if `wrapper.contains('some.translation.key')` instead of `expect(mocks.$t).toHaveBeenCalled('some.translation.key')`